packages/deployment/ansible/roles/cosmos-genesis/files/faucet-helper.sh

```diff
@@ -30,12 +30,12 @@ IFS=, read -r -a origRpcAddrs <<<"$(AG_SETUP_COSMOS_HOME=$thisdir ag-setup-cosmo
 
 rpcAddrs=(${origRpcAddrs[@]})
 while [[ ${#rpcAddrs[@]} -gt 0 ]]; do
-  r=$(( $RANDOM % ${#rpcAddrs[@]} ))
+  r=$(( RANDOM % ${#rpcAddrs[@]} ))
   selected=${rpcAddrs[$r]}
   rpcAddrs=( ${rpcAddrs[@]/$selected} )
 
   # echo "Checking if $selected is alive"
-  if [[ $(curl -s http://$selected/status | jq .result.sync_info.catching_up) == false ]]; then
+  if [[ $(curl -s http://"$selected"/status | jq .result.sync_info.catching_up) == false ]]; then
```

Comment:
you didn't happen to run this thru shellcheck, did you?

Reply:
> you didn't happen to run this thru shellcheck, did you?

I sure did.  Can you tell?